scout/server/blueprints/alignviewers/views.py

```diff
@@ -76,51 +80,44 @@ def unindexed_remote_static():
 
 
 @alignviewers_bp.route(
-    "/igv-splice-junctions/<institute_id>/<case_name>/<variant_id>", methods=["GET"]
+    "/<institute_id>/<case_name>/<variant_id>/igv-splice-junctions", methods=["GET"]
 )
 def sashimi_igv(institute_id, case_name, variant_id):
     """Visualize splice junctions on igv.js sashimi-like viewer for one or more individuals of a case.
     wiki: https://github.com/igvteam/igv.js/wiki/Splice-Junctions
     """
-    display_obj = controllers.make_sashimi_tracks(institute_id, case_name, variant_id)
-    return render_template("alignviewers/igv_sashimi_viewer.html", **display_obj)
-
-
-@alignviewers_bp.route("/igv", methods=["POST"])
-def igv():
-    """Visualize BAM alignments using igv.js (https://github.com/igvteam/igv.js)"""
-
-    # Set genome build for displaying alignments:
-    # Genome build is 37 if request.form.get("build") is 37 and chr != MT
-    # Genome build is 38 if request.form.get("build") is 38 or if chrom == MT
-    chromosome_build = request.form.get("build")
-    chrom = request.form.get("contig")
-    if chrom == "MT":
-        chrom = "M"
-    if chromosome_build in ["GRCh38", "38"] or chrom == "M":
-        chromosome_build = "38"
-    else:
-        chromosome_build = "37"
-
-    start = request.form.get("start")
-    stop = request.form.get("stop")
-    locus = "chr{0}:{1}-{2}".format(chrom, start, stop)
-
-    display_obj = {}  # Initialize the dictionary containing all tracks info
+    _, case_obj = institute_and_case(
```

Comment:
Passing through this function makes the code safe (checks that institute_and_case function takes care of checking if the user is allowed to see the resource. There is no additional check required. Check the code executed in this function: https://github.com/Clinical-Genomics/scout/blob/4c177c838639cb648bbe03c4be6f710f3dcc4d2c/scout/server/utils.py#L89

I've also written a test at this regard (if your user is not authorized then you get a 403 (unathorized error message)